seed/python-sdk/audiences/src/seed/core/client_wrapper.py

```diff
@@ -24,10 +26,10 @@ def get_base_url(self) -> str:
 class SyncClientWrapper(BaseClientWrapper):
     def __init__(self, *, base_url: str, httpx_client: httpx.Client):
         super().__init__(base_url=base_url)
-        self.httpx_client = httpx_client
+        self.httpx_client = HttpClient(httpx_client=httpx_client)
```

Comment:
nit, should we call this `self.client` now ?

Reply:
I kept it httpx_ since it's a thin wrapper and we use client so much that I wanted it to have some prefix haha